### PR TITLE
action intent example

### PR DESCRIPTION
**Pull request by vkosuri**
I think having this example would be useful Ref: https://github.com/gunthercox/ChatterBot/issues/820

### CODE REVIEW

File: examples/action_intent_example.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    'No, I have not',
+    'This should help get you started: http://chatterbot.rtfd.org/en/latest/quickstart.html'
+])
+
+# Get a response from bot
+response = bot.get_response('current weather in London?')
+# Handle JSON data, get all required data and print them
+if 'http' in response.text:
+    resp = requests.get(response.text)
+    print resp.json()
```
Comment (gabru-md) [inline examples/action_intent_example.py]: Build fails here at line 41 due to syntax error encountered in Python 3.
Following changes can help:
```python
from __future__ import print_function
print(resp.join())
```
Thanks!!

**Comment by vkosuri** [comment]
👍

**Comment by nggary** [comment]
@gunthercox Hi, will you do a enhancement for this purpose?

**Comment by gunthercox** [comment]
@nggary I'm not planning on doing one.

**Pull request closed without merging**